instrumentation/github.com/aws/aws-sdk-go/service/otels3/config.go

```diff
@@ -13,63 +13,63 @@
 // limitations under the License.
 
 // Package config sets options for the aws sdk instrumentation
-package config
+package otels3
 
 import (
 	"go.opentelemetry.io/otel"
 	otelmetric "go.opentelemetry.io/otel/api/metric"
 	oteltrace "go.opentelemetry.io/otel/api/trace"
 )
 
-// Config provides options for the aws sdk instrumentation.
-type Config struct {
-	TracerProvider           oteltrace.TracerProvider
-	MetricProvider           otelmetric.MeterProvider
-	Propagators              otel.TextMapPropagator
-	SpanCorrelationInMetrics bool
+// Config provides options for the AWS SDK instrumentation.
+type config struct {
+	TracerProvider  oteltrace.TracerProvider
+	MetricProvider  otelmetric.MeterProvider
+	Propagators     otel.TextMapPropagator
+	SpanCorrelation bool
 }
 
 // Option interface used for setting instrumentation configuration options.
 type Option interface {
```

Comment:
if the only method is private, should we also change the interface itself to private?

Reply:
we need to be able to reference the type when declaring the types. Also golang complains that the function is exported but returns an unexported type.